psi4/src/psi4/mrcc/mrcc.cc

```diff
@@ -640,8 +642,11 @@ PsiReturnType mrcc_load_ccdensities(SharedWavefunction wave, Options &options, c
     if (options.get_str("REFERENCE") == "UHF")
         restricted = false;
 
-    if (pertcc && options.get_str("REFERENCE") == "ROHF")
-        throw PSIEXCEPTION("Perturbative methods not implemented for ROHF references.");
+    if (pertcc && options.get_str("REFERENCE") == "ROHF") {
+        outfile->Printf("\n");
+        outfile->Printf("  WARNING: ROHF references are not implemented for perturbative\n");
+        outfile->Printf("           methods on older versions of MRCC. Proceed with caution.\n\n");
+    }
```

Comment:
MRCC-2014-07-01 and newer versions appear to have this ability.  I can't speak for older versions.